docs/user/whats-new.asciidoc

```diff
@@ -39,6 +39,87 @@ image::images/case-notification.png[Example email with case name, description, t
 
+[discrete]
+=== Dashboard
+
+[discrete]
+==== Wildcard search in dashboard controls
+**Dashboard** now supports searching for strings that are found within the values shown in a control. Previously, it was required for the string to be located at the start of the value in order to be found. However this is a more expensive option then just searching for the prefix, so it’s disabled by default. To enable it in **Dashboard** select Edit control > Options list settings > Contains. 
+
+[role="screenshot"]
+image::images/wildcard-search.png[Example of new string search]
+
+[discrete]
+=== Lens
+
+[discrete]
+==== Stacked bar chart legend
+
+The order of stacked bar chart legends is now based on popularity with the most important element displayed on top. You no longer need to scroll to the bottom of the legend to find the most popular data in the chart. 
+
+Previously:
+[role="screenshot"]
+image::images/oldwayhighlighted.png[Example of the old ordering of the chart legend, width=80%]
 
+New in 8.9:
+[role="screenshot"]
+image::images/newwayhighlighted.png[Example of the new ordering of the chart legend, width=80%]
+
+[discrete]
+==== Step curve
+
+**Lens** now supports three new options for line charts. The three options are straight, smooth, and step. Step curve line charts help users see clearly when data changes at irregular intervals, which helps to avoid misinterpretation. When creating visualizations select **Line** in the visualization type dropdown. Then use the visual options menu to choose between straight, smooth, or step in the Line interpolation dropdown. 
+
+[role="screenshot"]
+image::images/stepcurve.gif[An short video displaying the three new chart options]
 
+[discrete]
+==== Significance (significant terms)
+
+This new ranking method in the top values breakdown displays data by significance (significant terms). It permits us to get the terms that are important and likely the most critical for a given query. It’s a powerful way to find anomalies in your data with multiple use cases. Read more {ref}/search-aggregations-bucket-significantterms-aggregation.html[here].
+This is not just based on the most popular terms in a set. When calculating significant terms, Elasticsearch looks for significant changes in popularity of a specific term between two sets of data. A foreground set (the data returned by a query) and a background set (the data in our indices). This is supported on all charts. 
+
+[role="screenshot"]
+image::images/significant-terms.png[Example of a donut chart organized by significant terms, width=70%]
+
+[discrete]
+==== Tag cloud
 
+You can now create a word cloud in **Lens** using Tag cloud, which is currently in technical preview. When creating new visualizations, select **Tag cloud** in the visualization type dropdown. From the available fields drag and drop the desired field into the workspace to visualize keywords. 
```

Comment:
For items in technical preview, we should include a `preview:[]` admonition, similar to the ML one below.